.gitignore

```diff
@@ -5,3 +5,7 @@ Gemfile.lock
 *.gem
 .idea
 .DS_Store
+
+/.vs/VSWorkspaceState.json
```

Comment:
just ignore the whole `.vs` folder, same as the `.idea` is ignored above. no need to exclude specific files. also this path is absolute (starts with /) which wont work. these need to be relative to the repo